spring-web/src/main/java/org/springframework/web/cors/CorsUtils.java

```diff
@@ -38,19 +39,22 @@ public abstract class CorsUtils {
 	 * Returns {@code true} if the request is a valid CORS one by checking {@code Origin}
 	 * header presence and ensuring that origins are different.
 	 */
-	public static boolean isCorsRequest(HttpServletRequest request) {
+	public static IsCorsRequestResult isCorsRequest(HttpServletRequest request) {
```

Comment:
Is it OK to introduce a breaking public API change for this bugfix?  If so, it's probably worth documenting in the release notes.